<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>561</width>
     <height>371</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Downloads/pexels-pixabay-247676.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>521</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Script</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">.QLabel {
color: blue
}</string>
   </property>
   <property name="text">
    <string>Are You Sure You Want To Begin?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="yes_btn">
   <property name="geometry">
    <rect>
     <x>274</x>
     <y>187</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Print</family>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">.QPushButton { 
background: blue;
border-radius: 10px;
color: white
}

.QPushButton:hover { 
background: white;
border-radius: 10px;
font-weight: bold;
color: black;
}</string>
   </property>
   <property name="text">
    <string>Yes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="no_btn">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>187</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Print</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">.QPushButton { 
background: blue;
border-radius: 10px;
color: white
}

.QPushButton:hover { 
background: white;
border-radius: 10px;
font-weight: bold;
color: black;
}</string>
   </property>
   <property name="text">
    <string>No</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
